_episodes/01-remotes.md

```diff
@@ -1,20 +1,19 @@
 ---
 layout: episode
-title: Working with remotes
+title: Copying, sharing, and synchronizing repositories
 teaching: 20
 exercises: 0
 questions:
+  - How can we backup repositories?
   - How can we share repositories with others?
   - How can we keep repositories in sync?
+  - What are different ways to make a copy of the entire repository?
 objectives:
-  - Understand the difference between local branch, origin/branch, and remote branch.
+  - Be able to decide whether to divide work at the branch level or at the repository level.
 keypoints:
-  - "`git clone` copies everything and sets some pointers to remember where the clone came from."
-  - You communicate commits with `git fetch`/`git pull` and `git push`.
-  - All other Git operations are offline - you can work on a plane while your coworker is on vacation in North Korea.
-  - "`origin` refers to where you cloned from (but you can relocate it)."
-  - "`origin/foo` is a read-only pointer to branch `foo` on origin."
-  - "`origin` pointers only move when you `git fetch`/`git pull` or `git push`."
+  - "`git clone` copies everything: all commits and all branches."
```

Comment:
```suggestion
  - "`git clone` copies everything: all commits and all branches."
  - Branches on the remove appear as (read-only) local branches with a prefix, e.g. `origin/master`.
```
I'm not sure if this makes it too many points, but if phrased simply I think this is an important point and a common question at the beginning.